src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -126,13 +126,14 @@ Telemetry
 --------------
 The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community.
 You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell.
-You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.
-
-Configuring...
+You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.</value>
```

Comment:
This split is reasonable. There are two distinct messages here.

Sometimes it gets problematic when excessive pursuit of DRY eliminates the context that a translator might need. e.g. "{0}. Something else. {1}" where {0} and {1} have only a couple of fixed possibilities and it would be better to translate each combination as a whole.